### ISSUE TITLE
windows: quarto fails on render and create-project if the quarto exe and the target have spaces in the path

### ISSUE THREAD

**Issue by jjallaire** [rstudio/rstudio#11804]
If the Quarto executable has spaces in its path (as would be the case for a system wide install of RStudio in `C:\Program Files`) then there are problems both:

1) Creating projects with spaces in their path; and
2) Rendering documents with spaces in their path

These may be two slightly different cases as I know the former uses console processes and the latter does not. This also could be a regression as I don't recall this ever being a problem during initial development (but it certainly may have been).

**Comment by mikebessuille** [comment]
This is a good candidate for an automated test once it's fixed

**Comment by jgutman** [comment]
Think this is a duplicate of #11792 which has a PR up already

**Comment by jonvanausdeln** [comment]
Verified on 2022.11.0-148
Windows 11, Ubuntu 22.04, MacOS 12.4

Tested with varied paths and filenames with varied amounts of space and characters in path.

**Comment by statisco** [comment]
I have the same error after months of this thread. I was using the Google drive all the time for all my quarto docs. Now, if there is a space in the file path, like it is in the Google Drive, I have this error... When I put the same file into the desktop, I don't... Just bad...

ERROR: NotFound: The system cannot find the file specified. (os error 2), remove 'test\mediabag'

NotFound: The system cannot find the file specified. (os error 2), remove 'test\mediabag'
    at Object.removeSync (deno:runtime/js/30_fs.js:157:9)
    at removeIfEmptyDir (file:///C:/PROGRA~1/Quarto/bin/quarto.js:5701:18)
    at renderCleanup (file:///C:/PROGRA~1/Quarto/bin/quarto.js:80577:5)
    at file:///C:/PROGRA~1/Quarto/bin/quarto.js:81111:46
    at withTiming (file:///C:/PROGRA~1/Quarto/bin/quarto.js:19624:25)
    at Object.complete (file:///C:/PROGRA~1/Quarto/bin/quarto.js:81111:13)
    at async Object.onPostProcess (file:///C:/PROGRA~1/Quarto/bin/quarto.js:86678:36)
    at async renderFiles (file:///C:/PROGRA~1/Quarto/bin/quarto.js:86649:13)
    at async render (file:///C:/PROGRA~1/Quarto/bin/quarto.js:90681:21)
    at async renderForPreview (file:///C:/PROGRA~1/Quarto/bin/quarto.js:102415:26)

**Comment by ronblum** [comment]
@statisco Thanks for letting us know! Is this happening when you try to render any Quarto file? What about R Markdown? Also, can you provide the following info?

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       :

**Comment by jgutman** [comment]
@statisco Could you also let us know if this works for you in filepaths with spaces in the path/filename that are not in Google Drive? e.g. for a file that is local, on a non-network drive?

**Comment by jonvanausdeln** [comment]
I just tested and I can't even save a file to Google Drive.  It works fine if it's a local HD and the path and/or filename have spaces.  This is not limited to quarto files. I will open a new issue
<img width="291" alt="image" src="https://github.com/rstudio/rstudio/assets/6433789/791469f6-6343-4a21-bac2-9dec2c4f5761">

**Comment by jonvanausdeln** [comment]
Sorry, I deleted my last comment as my setup was incorrect and I just fixed it.  I also can reproduce that rendering a document on the a Google Drive mapped drive gives the same error.

My google drive is mapped to `G:\My Drive` - that seems to be the default that Google Drive created when I installed it. I haven't tried changing it (or if that's even possible)

RMarkdown and R scripts work just fine. Only seems to be quarto documents
